TextInputKit/TextInputKit/Code/SwiftExtensions/CountableClosedRange+TextInputKit.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CountableClosedRange+Tests.swift
+//  TextInputKit
+//
+//  Created by Artem Starosvetskiy on 02/12/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+import Foundation
+
+extension CountableClosedRange {
```

Comment:
There are reasons not to do so:
- I don't have intention to support another library
- It is desired to provide the "simple to use" library. Every dependency of the library increases complexity (from the point of view of an unfamiliar library user).
- It is desired not to have even a possibility to conflict with other libraries (to cause naming collisions), contracting the API to fit the major mission of a library 😎.
It could be a good idea if the library was "in-company".